.github/workflows/clippy.yml

```diff
@@ -27,11 +27,13 @@ jobs:
           sudo apt-get update && \
           sudo apt-get install -y \
             libpam0g-dev \
-            libudev-dev \
+            libselinux1-dev \
             libssl-dev \
+            libsystemd-dev \
```

Comment:
I think we don't need this now? But I know there is some relationship to udev sometimes ....

Reply:
I think before it built without it, so it might not be needed anymore.